### PR TITLE
Beasta patch 2 - grammar fix for "reseted"

### PR DESCRIPTION
**Pull request by Beasta**
## Context
This fixes a simple grammar error. The past tense of reset is not "reseted" but reset. See [here.](https://www.wordhippo.com/what-is/the-past-tense-of/reset.html)

## Approach
I did a search of the entire repo and updated all instances of "reseted" to reset 

## Test instructions
I updated any instance of "reseted" in the tests as well

### CODE REVIEW

**Comment by alexpeterson91** [comment]
Thanks for pointing this out to us!  Our core developers are not native English speakers and therefore a lot of code developed early on (this is the repo to the backbone of DAppNode) has extensive grammatical errors and typos, but none of it affects the usability of the code.  I will review your changes and apply some of them.  There are some changes you made that need more than just a find/replace change due to different ways of the word being used which causes more grammar issues and what you pointed out aren't even the only typos in the file if you read it.  
Also some of them can cause issue with formatting that can then break the code.  Going forward please do not open PRs without first opening an issue pointing out the issue, then the team can easily apply fixes.
As the only native English speaker on the team one of my tasks is to do just that, however there are millions of lines of code in our repos and over 160 repos some with several readme files, so we just fix things as we come across them

Finally, we do not have the resources to go through and proofread all the comments in our code for typos and several of your commits are changes of comments in the code, its just not necessary especially when you ignore other errors in the same sentence.
Regardless thank you for your contribution but please always open and Issue first before a PR and **_NEVER_** use find/replace to change words in code, it's a recipe for disaster.

File: packages/admin-ui/src/layout.scss [hunk 1]
```diff
@@ -213,7 +213,7 @@ body {
   **************
   Safari has been reported to cause issues with a grid-area 
   responsive layout. When react re-painted the DOM, some
-  node's scroll position was reseted.
+  node's scroll position was reset.
```
Comment (alexpeterson91) [inline packages/admin-ui/src/layout.scss:216]:
```suggestion
  nodes' scroll position was reset.
```

File: packages/admin-ui/src/start-pages/ResetPassword.tsx [hunk 2]
```diff
@@ -71,7 +71,7 @@ export function ResetPassword({
         </Button>
       </InputForm>
 
-      {reqStatus.result && <Ok ok msg="Reseted password"></Ok>}
+      {reqStatus.result && <Ok ok msg="Reset password"></Ok>}
```
Comment (alexpeterson91) [inline packages/admin-ui/src/start-pages/ResetPassword.tsx:74]:
```suggestion
      {reqStatus.result && <Ok ok msg="Password Reset"></Ok>}
```

File: packages/dappmanager/src/modules/installer/packageIsInstalling.ts [hunk 3]
```diff
@@ -1,11 +1,11 @@
 /**
  * Stores the last unix timestamp a package was attempted to be installed
  * - If all go well, when an installation is completed this number
```
Comment (alexpeterson91) [inline packages/dappmanager/src/modules/installer/packageIsInstalling.ts:3]:
```suggestion
 * - If all goes well, when an installation is completed this number
```

File: packages/dappmanager/src/modules/installer/restartPatch.ts [hunk 4]
```diff
@@ -163,7 +163,7 @@ export async function postRestartPatch(): Promise<void> {
     const packagesDataRaw = db.coreUpdatePackagesData.get();
     const log = getLogUi(params.coreDnpName);
     if (!packagesDataRaw) {
-      // Assuming the dappmanager has been reseted, not updated
+      // Assuming the dappmanager has been reset, not updated
```
Comment (alexpeterson91) [inline packages/dappmanager/src/modules/installer/restartPatch.ts:166]:
```suggestion
      // Assuming the dappmanager has been reset, but not updated
```

File: packages/dappmanager/src/utils/autoUpdateHelper.ts [hunk 5]
```diff
@@ -223,7 +223,7 @@ export function clearPendingUpdates(dnpName: string): void {
 
 /**
  * Clears the pending updates from the registry so:
- * - The update delay time is reseted
+ * - The update delay time is reset
  * - The UI does no longer show the "Scheduled" info
```
Comment (alexpeterson91) [inline packages/dappmanager/src/utils/autoUpdateHelper.ts:227]:
```suggestion
 * - The UI no longer shows the "Scheduled" info
```

File: packages/dappmanager/src/modules/nsupdate/utils.ts [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import { isEmpty } from "lodash";
 import { PackageContainer } from "../../types";
-import { getContainerDomain } from "../../params";
+import params, { getContainerDomain } from "../../params";
```
Comment (alexpeterson91) [inline packages/dappmanager/src/modules/nsupdate/utils.ts:3]: these are functional changes, not typos must be split to multiple PRs and have an associated Issue

**Comment by alexpeterson91** [comment]
Just realized this is not even the same base branch, youre 30 commits behind the main branch so im just gonna close this PR, thanks for your contribution.

**Comment by Beasta** [comment]
I'll keep all that in mind. Still a bit of a novice dev obviously. Thanks for your work on Dappnode @alexpeterson91 ! I really do love mine.

**Review by alexpeterson91 — COMMENTED** [review]
closed for team review

**Pull request closed without merging**